packages/core/src/amazonq/lsp/lspController.ts

```diff
@@ -219,13 +221,15 @@ export class LspController {
         return true
     }
 
-    async tryInstallLsp(context: vscode.ExtensionContext): Promise<boolean> {
+    async tryInstallLsp(context: vscode.ExtensionContext, forceReset = false): Promise<boolean> {
```

Comment:
Can we make these changes on the `feature/amazonqLSP` branch instead? I think the resolve() function in `core/src/shared/lsp/lspResolver.ts` could have this instead. That way you could log the language server name in addition to that its downloading